test/integration/patients/worklist/schedule.js

```diff
@@ -1,6 +1,6 @@
 import _ from 'underscore';
 import dayjs from 'dayjs';
-import { v5 as uuid } from 'uuid';
+import { v4 as uuidv4, v5 as uuidv5 } from 'uuid';
```

Comment:
We don't do this anywhere else, where both v4 & v5 imported with different naming. I'm not sure how else to do it...